contrib/spec/podman.spec.in

```diff
@@ -3,6 +3,7 @@
 %global with_debug 1
 %global with_check 0
 %global with_unit_test 0
+%global with_doc 0
```

Comment:
Default should be true, to not break existing users.

Reply:
Done.

Reply:
Done, Thanks :-)